csrc/scheduler/matmul.cpp

```diff
@@ -931,7 +931,33 @@ void scheduleMatmul(Fusion* fusion, const MatmulParams& params) {
     }
   }
 
-  // [Mo, No, Ko, Mi, Ni, Ki]
+  // Split-K: We always split by the Split-K factor, but we don't parallelize
```

Comment:
Is this statement "We always split..." still true?

Reply:
Good catch. It is no longer true.